src/util.lisp

```diff
@@ -27,7 +27,7 @@
 (defun is-git-url? (str)
   "Is <str> a git:// or .git url."
   (or (ppcre:scan "^git://.*" str)
-      (ppcre:scan ".*.git" str)))
+      (ppcre:scan ".*\.git$" str)))
```

Comment:
This means that `https://github.com/CodyReichert/qi` no longer returns true.  I think that's the correct behavior.  If you you disagree, we could maybe change it to
```lisp
(defun is-git-url? (str)
  "Is <str> a git:// or .git url."
  (or (ppcre:scan "^git://.*" str)
      (ppcre:scan ".*\.git$" str)
      (is-gh-url? str)))
```
Which is more explicit.